app/src/main/java/org/oppia/app/player/audio/LanguageInterface.kt

```diff
@@ -7,4 +7,8 @@ interface LanguageInterface {
 
   /** Open the language selection dialog. */
   fun languageSelectionClicked()
+
+  fun getUserIsSeeking(): Boolean
```

Comment:
Similarly--please add documentation here & elsewhere when changing interfaces.

Reply:
Sure! Just out of curiosity, how come these two aren't considered "obvious"? I just thought since they were simple getters, they might be?